packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -118,9 +118,9 @@ class ExpiringMultiPartyClient {
     }
     // Fetch contract state variables in parallel.
     const [newSponsorEvents, endedSponsorEvents, liquidationCreatedEvents, currentTime] = await Promise.all([
-      this.emp.getPastEvents("NewSponsor", { fromBlock: 0 }),
-      this.emp.getPastEvents("EndedSponsorPosition", { fromBlock: 0 }),
-      this.emp.getPastEvents("LiquidationCreated", { fromBlock: 0 }),
+      this.emp.getPastEvents("NewSponsor", { fromBlock: await getFromBlock() }),
```

Comment:
why is this `await getFromBlock()` when `getFromBlock` is not async and simply returns an environment var?

Reply:
Good catch! It shouldn't be anymore. This is an artifact from when I was looking at querying the current block of whatever chain (Ganache, mainnet, etc.) but is no longer necessary. Will fix that.